services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -146,7 +146,11 @@ def create_base_app() -> FastAPI:
     override_fastapi_openapi_method(app)
     app.state.settings = app_settings
 
-    long_running_tasks.server.setup(app)
+    long_running_tasks.server.setup(
+        app,
+        redis_settings=app_settings.REDIS_SETTINGS,
+        namespace=f"dy_sidecar-{app_settings.DY_SIDECAR_RUN_ID}",
```

Comment:
so I guess this could be renamed as "redis_key_prefix" or "redis_namespace" or something similar.

Reply:
I left it vague since I tink it's also required in a future PR for Rabbit. But I will think about it, if it requires a better name

Reply:
redis_namespace in the end